specification/chaos/resource-manager/Microsoft.Chaos/preview/2022-07-01-preview/types/capabilityTypes.json

```diff
@@ -0,0 +1,103 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Capability Metadata Types",
+    "version": "2022-07-01-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "capabilityType": {
+      "type": "object",
+      "description": "Model that represents a Capability Type resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "systemData": {
+          "description": "The system metadata properties of the capability type resource.",
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the Capability Type resource."
+        },
+        "properties": {
+          "description": "The properties of the capability type resource.",
+          "$ref": "#/definitions/capabilityTypeProperties",
+          "x-ms-client-flatten": true,
+          "readOnly": true
+        }
+      }
+    },
+    "capabilityTypeProperties": {
+      "type": "object",
+      "description": "Model that represents the Capability Type properties model.",
+      "properties": {
+        "publisher": {
+          "type": "string",
+          "description": "String of the Publisher that this Capability Type extends.",
+          "readOnly": true
+        },
+        "targetType": {
+          "type": "string",
+          "description": "String of the Target Type that this Capability Type extends.",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Localized string of the display name.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "Localized string of the description.",
+          "readOnly": true
+        },
+        "parametersSchema": {
+          "$ref": "./common.json#/definitions/url",
+          "description": "URL to retrieve JSON schema of the Capability Type parameters.",
+          "readOnly": true
+        },
+        "urn": {
+          "$ref": "./common.json#/definitions/urn",
+          "description": "String of the URN for this Capability Type.",
+          "readOnly": true
+        },
+        "kind": {
+          "type": "string",
+          "description": "String of the kind of this Capability Type.",
+          "readOnly": true
+        },
+        "runtimeProperties": {
+          "$ref": "./common.json#/definitions/runtimeProperties",
+          "description": "Runtime properties of this Capability Type.",
+          "readOnly": true
+        }
```

Comment:
If `kind` and `runtimeProperties` are the only changes to this PR, could we just re-use the existing API version (`2021-09-15-preview`) since this wouldn't be a breaking change?

Reply:
**[noop]** What are the possible values for `kind`?